### DISCUSSION TITLE
Newer versions aren't offered in Google Play Store

### DISCUSSION THREAD

**Discussion by feuerwasser85** [astubenbord/paperless-mobile#225] [Q&A] [answered]
Why am i getting only 2.3.7 offered in Google Play Store?
I have a Samsung SM-S908B with the latest Android on it.
Is there anything i am missing?

**Comment by astubenbord** [comment, marked as answer]
You can join the open beta to get them. But they've been out long enough to be released to production, so I'll publish them. You can still join the beta to always get the newest releases :)

**Reply by feuerwasser85** [reply]
I still see 2.3.7 in the PlayStore. Did you already publish 2.3.9?

**Comment by feuerwasser85** [comment]
Thanks for the (lightning)-fast response!
Am i able to see the latest stable releases in https://github.com/astubenbord/paperless-mobile/releases? v2.3.9 is currently marked as Latest but not in PlayStore.

**Comment by astubenbord** [comment]
I just released the newest version to production.

**Reply by feuerwasser85** [reply]
I'm sorry, i don't get it. How can i get 2.3.9 in the playstore? I still get 2.3.7.

**Reply by astubenbord** [reply]
It'll take some hours for Google to approve, should be up today :)

**Comment by Diesmo** [comment]
Google Playstore still offers only 2.3.9 as latest version.

The latest Github release says 3.1.1. Any chance that we can get the latest Github version via Google Playstore?

**Reply by astubenbord** [reply]
It's still in beta but I'll release it for everyone soon. You can join the open beta on Google play to always get the latest releases though :)
